bitcoin/src/psbt/macros.rs

```diff
@@ -9,6 +9,52 @@ macro_rules! hex_psbt {
     };
 }
 
+#[cfg(test)]
+macro_rules! psbt_with_values {
```

Comment:
Can we achieve similar behavior with a simple function?

Reply:
Yes. I was merely following what was already in the tests.

Where should we put helper functions for the tests?

I'm not sure if this is something we should include in this PR, but I definitely agree that all these helper functions/macros should be in a more appropriate place.

Reply:
I think the location is fine; the question is just whether we want to use a macro or a function.

Reply:
But I agree that we can leave this for a followup PR (and that there are probably other instances we should fix at the same time)

Reply:
I think it's preferable here to just use a regular function here since there is no reason to use pre-processing.  If it's not a macro, then probably don't put the function in `macros.rs`.  Instead I would just put the helper function in mod.rs in the tests module.